.claude/hooks/utils/llm/oai.py

```
@@ -0,0 +1,172 @@
+# dependencies = [
+#     "openai",
+#     "python-dotenv",
+# ]
+# ///
+
+import os
+import sys
+from dotenv import load_dotenv
+
+
+def prompt_llm(prompt_text):
```

Ruff (0.12.2)

`prompt_llm` is too complex (12 > 10)

(C901)